Find a functionally equivalent code:com.android.internal.widget.LockPatternUtils.getActivePasswordQuality:COMMENT
Method Modifier: public      internal    
Comment:/**
 * Used by device policy manager to validate the current password
 * information it has.
 */

@@ -19,24 +19,29 @@
                 activePasswordQuality = DevicePolicyManager.PASSWORD_QUALITY_NUMERIC;
             }
             break;
+        case DevicePolicyManager.PASSWORD_QUALITY_NUMERIC_COMPLEX:
+            if (isLockPasswordEnabled()) {
+                activePasswordQuality = DevicePolicyManager.PASSWORD_QUALITY_NUMERIC_COMPLEX;
+            }
+            break;
         case DevicePolicyManager.PASSWORD_QUALITY_ALPHABETIC:
             if (isLockPasswordEnabled()) {
                 activePasswordQuality = DevicePolicyManager.PASSWORD_QUALITY_ALPHABETIC;

